_posts/2024-01-16-log-detective.md

```diff
@@ -0,0 +1,67 @@
+---
+title: AI-driven Build Failure diagnosis?
```

Comment:
I'm not sure about the question mark at the end.

Can you please explain to me why the title looks like a question?